tests/test_cyto_utils/test_collate.py

```diff
@@ -9,22 +9,31 @@
 
 plate = "BR00121431"
 
-test_config_location = pathlib.Path(
-    f"{os.path.dirname(__file__)}/../../cyto_utils/database_config/ingest_config.ini"
+test_dir = pathlib.Path(__file__).parents[1].absolute()
```

Comment:
Consider using all-caps for global variable names in this file to help distinguish these. Alternatively, consider using a [Pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) which would help here and possibly elsewhere along with reducing repetition. For example:
```suggestion
TEST_DIR = pathlib.Path(__file__).parents[1].absolute()
```

Reply:
Done.